src/Engine/ProtoCore/Lang/Replication/Replicator.cs

```diff
@@ -337,7 +337,52 @@ public static List<StackValue> EstimateReducedParams(List<StackValue> formalPara
 
             return reducedParamTypes;
         }
+        
+        /// <summary>
+        /// Compute the effects of the replication guides on the formal parameter lists
+        /// The results of this loose data, and will not be correct on jagged arrays of hetrogenius types
+        /// </summary>
+        /// <param name="formalParams"></param>
+        /// <param name="replicationInstructions"></param>
+        /// <returns></returns>
+        internal static List<CLRStackValue> EstimateReducedParams(List<CLRStackValue> formalParams, List<ReplicationInstruction> replicationInstructions)
```

Comment:
These "formalParams" are the arguments, correct?

Reply:
YEs...I kept the name from the old vm function.
(as a general rule, if the array type is CLRStackValue...then it represents the arguments)